_data/sites.yml

```
@@ -1382,6 +1387,11 @@
   url: https://herman.bearblog.dev/
   size: 6.5
   last_checked: 2022-09-12
+  
+- domain: hispagatos.org
+  url: https://hispagatos.org
+  size: 256
```

Uncompressed size should be used.

```suggestion
  size: 405
```